### PR TITLE
Orquesta-ifying the pipeline workflow.

### PR DESCRIPTION
**Pull request by mjtice**
Since the plan is to deprecate (or it is now I guess) mistral support, I thought this should be orquesta-ified.

### CODE REVIEW

File: pack.yaml [hunk 1]
```diff
@@ -5,6 +5,6 @@ runner_type: "python-script"
 description: Terraform integrations
 keywords:
     - terraform
-version: 0.2.6
+version: 0.2.7
```
Comment (arm4b) [inline pack.yaml]: According to https://semver.org/, I think the version should be `0.3.0` since it's not a bugfix, but rather a change

**Review by arm4b — CHANGES_REQUESTED** [review]
That's helpful! Thanks for your work migrating it to Orquesta :+1: 

I just left one minor comment

File: pack.yaml (hunk 1 above)
Comment (punkrokk) [reply pack.yaml]: I almost typed that same thing @blag said from my phone

Comment (mjtice) [reply pack.yaml]: ok cool, I've updated the pr.

**Review by arm4b — APPROVED** [review]
Thank you @mjtice  :+1:

**Pull request merged**